lib/cl.js

```diff
@@ -0,0 +1,17 @@
+// Process Command Line Arguments
+
+// At this stage we only support a few flags,
+// so it's OK to process them one by one like this
+// If one day we need to process more command line arguments,
+// it's better to find a 3rd party command line library to do this job.
+process.argv.forEach(function(arg) {
+  "use strict";
+  if (/^-(-debug|d)$/i.test(arg)) {
+    process.env.DEBUG = 1;
+    console.log("Debug Mode");
+  } else if (/^--debug-plugin=\w[\w-]*$/i.test(arg)) {
+    let name = arg.split("=")[1];
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).